.github/workflows/ci.yml

```diff
@@ -69,6 +69,9 @@ jobs:
           pip3 install --pre torch -f "${{ steps.pytorch_channel.outputs.value }}"
           pip3 install cmake ninja
           echo "/home/runner/.local/bin" >> $GITHUB_PATH
+      - name: Run AIStore local deployment
+        if: matrix.os != 'windows-latest'
+        uses: NVIDIA/aistore@master
```

Comment:
**Running AIStore as a Docker Container**
Generally, AIStore is a storage cluster that requires a volume (ability to mount using `-v` in docker). In Github workflow environments, when we try to mount the `dev/root` as a volume to the container we receive an error during the startup which says that `dev/root` doesn't have underlying block storage. We can probably overcome this by running it from `/` or `/tmp` but we at AIStore don't recommend this. Currently, we are running it as a local deployment from scratch as a [Github action](https://github.com/NVIDIA/aistore/blob/master/action.yml).
**Running AIStore in different pipeline**
We can run AIStore as a separate workflow and import it into the CI workflow to run the python tests. But then we need to have a reliable mechanism to synchronize the testing pipeline with the AIS cluster starting up (in a separate pipeline). The latter may take anywhere between tens of milliseconds to maybe 10-20s depending on the load of the machine we run in. I have tried to simulate the [torch/data pipeline of this PR](https://github.com/gaikwadabhishek/data/runs/7032258015). 
 
Let me know what you guys think, we can work on this.

Reply:
The reason I want to have a separate workflow is we can execute tests for AIStore within that workflow without touching others. With separate workflow, I am able to disable the test easily using GHA whenever we find constant failing tests.

Reply:
> We can run AIStore as a separate workflow and import it into the CI workflow to run the python tests.

You don't need to do this. Since you AIStore doesn't require AWSSDK installed, you can basically copy paste the most of parts from `ci.yml` by removing all AWS related steps like:
- https://github.com/pytorch/data/blob/75f31dc99dcc3159ee9a3df73d3862c76754823c/.github/workflows/ci.yml#L44-L49
- https://github.com/pytorch/data/blob/75f31dc99dcc3159ee9a3df73d3862c76754823c/.github/workflows/ci.yml#L54-L61
- https://github.com/pytorch/data/blob/75f31dc99dcc3159ee9a3df73d3862c76754823c/.github/workflows/ci.yml#L70-L71
- Remove BUILD_S3=1. https://github.com/pytorch/data/blob/75f31dc99dcc3159ee9a3df73d3862c76754823c/.github/workflows/ci.yml#L75-L76
- Modify the tests steps to only execute the tests for AIStore.

Reply:
Okay, I have made the changes. Do you want to trigger this workflow on a specific label? Currently the condition is similar to the main `ci.yml` -
```if:
      ${{ github.repository_owner == 'pytorch' && (github.event.action != 'labeled' ||
      startsWith(github.event.label.name, 'ciflow')) }}```